src/scenes/Wallet/components/ActionButtonHeader/index.js

```diff
@@ -40,25 +38,13 @@ const messages = defineMessages({
   },
 });
 
+@inject('store')
+@observer
```

Comment:
move `inject('store') and observer` to the very last injections. this can sometimes cause really weird bugs if its above others.